packages/jest-cli/src/watch.js

```diff
@@ -400,6 +401,7 @@ export default function watch(
   };
 
   if (typeof stdin.setRawMode === 'function') {
+    // $FlowFixMe
```

Comment:
mind adding a comment on why `setRawMode()` causes problems?

Reply:
I'm not sure 😅 IDK why the `typeof` check isn't enough.

The error is: `Cannot call stdin.setRawMode because the parameter types of an unknown function [1] are unknown.`